deploy/vagrant/vbox/buildbox.sh

```diff
@@ -12,9 +14,16 @@ if [ -f tachyon-dev.box ]; then
   rm -f tachyon-dev.box
 fi
 
-vagrant up
-vagrant package --output tachyon-dev.box default
-vagrant destroy -f
+BASE_BOX='chef/centos-7.0'
+AVAIL_BOX=$(vagrant box list | grep $BASE_BOX | grep '(virtualbox, 1.0.0)')
+
```

Comment:
Thanks for the explanation!

Yes, I agree having a tachyon-dev box with java/maven/ etc installed will be launched faster..I thought that's what the old tachyon-dev box is, after running the provision.sh script. 

The new tachyon-dev box does not seem to contain any of those dependencies. It looks like a vanilla centos box, which is why I am a little confused. 
Would a better fix to TACHYON-804 be to skip ansible tasks if java/maven is already installed? (i.e. it came with the image, tachyon-dev.box that we built)